src/Components/test/testassets/BasicTestApp/BindGenericComponent.razor

```diff
@@ -0,0 +1,9 @@
+@typeparam TValue
+
+<input id="@Id" type="text" value="@Value" @onchange="@(EventCallback.Factory.CreateBinder<TValue>(this, _ => ValueChanged.InvokeAsync(_), Value))" />
```

Comment:
The manual use of `CreateBinder` here looks pretty hardcore. Is your expectation that customers will do this? I'm not sure I'd feel confident writing this code and being sure it was right without copy/pasting some example.

Out of interest, since I can't run this code right now, does the following simplified syntax (relying on generic type inference and methodgroup-to-delegate conversion) produce the same result?

```razor
<input id="@Id" type="text" value="@Value" @onchange="@(EventCallback.Factory.CreateBinder(this, ValueChanged.InvokeAsync, Value))" />
```

Reply:
I'll try this and see. I expect it does.

Reply:
It does not work because all of these methods accept `Action<>`